app/models/user.rb

```diff
@@ -46,6 +48,30 @@ def events
     accepted_applications.collect { |a| a.event }
   end
 
+  # Returns true iff. user has submitted an agreement_letter for the given event
```

Comment:
Please annotate your public methods in YARD syntax (http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md)